tests/e2e/utils/helpers.py

```diff
@@ -55,28 +63,61 @@ def get_logs(lambda_function_name: str, log_client: CloudWatchClient, start_time
     return filtered_logs
 
 
-@lru_cache(maxsize=10, typed=False)
-@retry(ValueError, delay=1, jitter=1, tries=20)
+@retry(ValueError, delay=1, jitter=1, tries=10)
```

Comment:
In some rare occasions the 10 tries was not enough - thats why I raised this number to 20

Reply:
good point, I decreased because it took forever to fail. Let's meet in the middle by increasing the jitter to 1.5